schema/queue/message/impl/RabbitMqMessageSubscriber.go

```diff
@@ -48,6 +58,7 @@ type CommonMessageSubscriber struct {
 }
 
 func (cs *CommonMessageSubscriber) Handler(msgDelivery amqp.Delivery) {
+	startTime := time.Now()
```

Comment:
Could please have a look at the https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Timer